src/control/server/storage/faults.go

```diff
@@ -60,15 +60,23 @@ func FaultScmInvalidPMem(msg string) *fault.Fault {
 }
 
 // FaultScmNoModules represents an error where no PMem modules exist.
-var FaultScmNoModules = storageFault(code.ScmNoModules,
-	"No PMem modules exist on storage server",
-	"Install PMem modules and retry command")
-
-// FaultBdevConfigTypeMismatch represents an error where an incompatible mix of emulated and
+var FaultScmNoModules = storageFault(
```

Comment:
Nit, the following error name could be more comprehensible:
```suggestion
var FaultScmNoPmemModules = storageFault(
```

Reply:
done